<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoneShoppingGui</class>
 <widget class="QDialog" name="DoneShoppingGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dokan Online Market</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/img/assets/img/icons/icons8-small-business-96.png</normaloff>:/img/assets/img/icons/icons8-small-business-96.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color: #e6e6e6;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>511</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/img/assets/img/icons/shopping cart shopping bag.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
background-color: rgba(0,0,0,0);
color: #ca9802;
}</string>
   </property>
   <property name="text">
    <string>We hope you enjoyed your shopping with us  - Have a nice day ‚ù§</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>521</width>
     <height>125</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lb_date">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: rgba(0,0,0,0);
color: #2679a6;
}</string>
      </property>
      <property name="text">
       <string>Your order will arrive on 8 may</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_address">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: rgba(0,0,0,0);
color: #2679a6;
}</string>
      </property>
      <property name="text">
       <string>At ain shams street</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_price">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: rgba(0,0,0,0);
color: #990000;
letter-spacing: 1px;
}</string>
      </property>
      <property name="text">
       <string>Total :  1504 EGP</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
